packages/protocol/contracts/governance/ReleaseGold.sol

```diff
@@ -131,8 +137,26 @@ contract ReleaseGold is UsingRegistry, ReentrancyGuard, IReleaseGold, Initializa
     _;
   }
 
+  modifier onlyExpired() {
+    uint256 releaseEndTime = releaseSchedule.releaseStartTime.add(
+      releaseSchedule.numReleasePeriods.mul(releaseSchedule.releasePeriod)
+    );
+    require(
+      block.timestamp >= releaseEndTime.add(EXPIRATION_TIME),
+      "`EXPIRATION_TIME` must have passed after the end of releasing"
+    );
+    _;
+  }
+
   function() external payable {} // solhint-disable no-empty-blocks
 
+  /**
+   * @notice Wrapper function for stable token transfer function.
+   */
+  function transfer(address to, uint256 value) external onlyWhenInProperState {
+    IERC20(registry.getAddressForOrDie(STABLE_TOKEN_REGISTRY_ID)).transfer(to, value);
```

Comment:
Does `getStableToken` from `UsingRegistry` not work?

Reply:
Unfortunately no, `getStableToken` returns `IStableToken` which doesn't expose the `transfer` fn. Since `StableToken` is ERC20 compliant, casting it to `IERC20` is safe here.
Asa and I decided to just use the typecast rather than expose transfer on `IStableToken`, if you think it's better to add to the interface let me know